src/front/app/__test__/HelloWorld.test.js

```diff
@@ -0,0 +1,3 @@
+test('Test JEST suite base functionality', () => {
+  expect('Hello World').toBe('Hello World');
+});
```

Comment:
Important, this is alright since you don't have anything to test, but how these are made is:
```js
describe('(Component) MyComponent', () => {
   it('does whatever you expect', () => {
       expect(MyComponent).toDo('something');
   });
});
```